### PR TITLE
Update 01-intro.md

### PR DESCRIPTION
**Pull request by RachelMc-dev**
Add clearer definition of what a variable is, this could be done somewhere around line 19. For example: Variables are used to store information to be referenced and manipulated in a computer program. It is helpful to think of variables as containers that hold information. Their sole purpose is to label and store data in memory. This data can then be used throughout your program.

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

**Comment by maxim-belkin** [comment]
Hi @RachelMc-dev. Thank you for your contribution!

1. What was your motivation for suggesting this clarification?
2. SWC workshops are hands-on workshops in which coding sessions are interspersed with explanations. These explanations are closely connected to the code participants see. Your explanation is somewhat abstract and has a signitifant cognitive load (you mention "variables", "information", "data", "computer program", "labels", "containers", "memory" not to mentions connections between these). It has to be simplified.
3. Related to the points above and Lauren's comment: we can't start with this explanation. A better place for it would be after the `In layman's terms, **a variable is a name for a value**.` sentence.

**Comment by RachelMc-dev** [comment]
Hi,

When I taught this section of Python it was evident that the information provided was more focused on explaining variables from a data basis rather than a programming basis, a colleague pointed it out in feedback that the carpentries wording was ambiguous.  I really don’t mind if it’s not changed I just wanted to provide some feedback and it was the only section of the python material that caused confusion.  I’m happy to just explain in more next time I teach.  Cheers, Rachel.
